src/util/errors.js

```diff
@@ -136,6 +136,23 @@ export const isTransactionInitiateAmountTooLowError = error => {
   });
 };
 
+/**
+ * Check if the given API error (from `sdk.transaction.initiate()`) is
+ * due to other error in Stripe.
+ */
+export const isTransactionInitiateStripeError = error => {
```

Comment:
Since this is not returning any boolean, the name should be something else. 
How about `transactionInitiateOrderStripeErrors` or `transactionInitiateStripeErrors`?